package.json

```diff
@@ -60,7 +60,7 @@
     "typescript": "^4.1.2"
   },
   "engines": {
-    "node": ">=12"
+    "node": ">=14.16"
```

Comment:
Is there something specifically added for the `x.16` or can `>=14` work?

Reply:
No, nothing special, just being more explicit as of "today", but I haven't tested with 14.1 for instance.